microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/java/definition/AbstractAnnotationDefinitionParticipant.java

```diff
@@ -119,12 +122,32 @@ public boolean isAdaptedForDefinition(JavaDefinitionContext context, IProgressMo
 		// Try to get the annotation member value
 		String annotationSource = ((ISourceReference) annotation).getSource();
 		String annotationMemberValue = null;
+		Range propertyKeyRange = null;
+		boolean found = false;

```

Comment:
I noticed that we have the same code in https://github.com/eclipse/lsp4mp/blob/ec5f1ab23b1d3f08f2bd6b60f37bea15a3069d6d/microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/java/hover/PropertiesHoverParticipant.java#L152 

We should provide an utility method in AnnotationUtils like 

getAnnotationMemberValueAt(Position position) which must return the propertyKeyRange and perhaps the  value as string (for hover). 

hover participant and definition participant should consumes getAnnotationMemberValueAt.

@rgrunber do you know if it's possible to get the range of member value to avoid using regexp?